packages/e2e/test/cip30.test.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @jest-environment jsdom
+ */
+
+//import { testnetTimeSettings } from '@cardano-sdk/core';
+//import { createStubStakePoolSearchProvider, createStubTimeSettingsProvider } from '@cardano-sdk/util-dev';
+//import { createWalletApi, injectWindow, WalletApi, Wallet } from '../../cip30';
+import { handleMessages, injectWindow, Wallet as BigWallet } from '../../cip30/src';
```

Comment:
```suggestion
import { handleMessages, injectWindow, Wallet as BigWallet } from '@cardano-sdk/cip30';
```

Setup a `tsconfig.json` like here: https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/wallet/test/tsconfig.json to establish the TypeScript dependencies. This will build the dependent code when needed.